### ISSUE TITLE
Return URL after Login?

### ISSUE THREAD

**Issue by jeffsikes** [flipboxfactory/saml-sp#11]
How do you configure the SSO process to send you back to the same page?

Example - I click "Login" on https://domain.com/puppies.  After the SSO process completes, I want it to return to /puppies, not the root of the site.

**Comment by dsmrt** [comment]
Hi @jeffsikes, 

Is your loginPath configured like suggested in the [docs](https://flipboxfactory.github.io/saml-sp/configure/login/#initiate-login-with-the-idp)? Like so (in your `general.php`):
```php
return [
  //...
  'loginPath' => '/sso/login/request',
  //...
];
```

I'm assuming it is, I just want to make sure.

**Comment by dsmrt** [comment]
I patched this with `1.0.0-beta.10`. Give it a try and let me know if you have any issues with this or anything else.

**Comment by jeffsikes** [comment]
My login path was as suggested in the docs, and I've verified it's working as expected for me.  Thanks!
